lib/inline_svg/action_view/helpers.rb

```diff
@@ -12,7 +12,7 @@ def inline_svg(filename, transform_params={})
             configured_asset_file.named filename
           end
         rescue InlineSvg::AssetFile::FileNotFound
-          return "<svg><!-- SVG file not found: '#{filename}' #{extension_hint(filename)}--></svg>".html_safe
+          return "<svg class='svg-not-found'><!-- SVG file not found: '#{filename}' #{extension_hint(filename)}--></svg>".html_safe
```

Comment:
We could add this configuration to the [`InlineSvg::Configuration` class](https://github.com/jamesmartin/inline_svg/blob/eee2360c6f1afe95edb7edd0bbbbe41990df616f/lib/inline_svg.rb#L65). Is that the kind of configuration you were thinking?

We could use it like this:

```ruby
InlineSvg.configure do |config|
  config.svg_not_found_css_class = "my-custom-class"
end
```